packages/editor/src/editor-styles.scss

```diff
@@ -12,8 +12,8 @@ p {
 
 ul,
 ol {
-	margin: 0;
-	padding: 0;
+	margin: 1em 0;
```

Comment:
Let's discuss this one too.

In browsers, if you don't load a stylesheet, `ol` and `ul` are both born with 1em margins above and below.

In this case, and the reason this rule sits in `editor-styles.scss`, is that it actually has to be rewritten again, otherwise the editing canvas will inherit upstream rules from the WordPress admin CSS.

While the long term solution here is to refactor the WordPress admin to be better scoped and not bleed into the editing canvas, that's likely a ways out. In the mean time, there's this.

Which brings us to the question: is this the right place for it? How about `theme.scss` like the `p` rules?

Reply:
This feels like this is needed to ensure the core markup in the editor renders in a suitable fashion out of the box. Therefore in my mind it should be part of the core editor styles rather than an optional Theme specific style. If Theme authors want to tweak the spacing they can do so or they can opt out of the `theme.css` file entirely and provide their own rules.

Reply:
Don't have _any_ styles on plain elements. If you can't put a class on it for the editing canvas, you shouldn't be styling it.

Reply:
> it should be part of the core editor styles rather than an optional Theme specific style.

I agree. This shouldn't be optional if we're re-normalizing something from `common.css` or an inline style from WP admin. If that's the case `initial` would work as well.

However, currently `theme.css` isn't optional in the editor so there would be fine if that doesn't change.

Reply:
> Don't have any styles on plain elements. If you can't put a class on it for the editing canvas, you shouldn't be styling it.

This is a noble goal, one we should strive for. But until we can prevent CSS bleed from wp-admin through, for example, Shadow DOM, we have to unstyle even plain elements, otherwise they will look like just another wp-admin settings page. Just to be completely clear, here's what that would look like: https://user-images.githubusercontent.com/1204802/54341682-b75ec680-463a-11e9-964f-607f5223364f.gif